### ISSUE TITLE
Homebrew-installed Elasticsearch fails on startup

### ISSUE THREAD

**Issue by dcalixto** [elastic/elasticsearch#14424]
after update to 2.0 did not start anymore now shows:

**es.config is no longer supported. elasticsearch.yml must be placed in the config directory and cannot be renamed.**

i'm using homebrew, someone get the same issue?

**Comment by ssoulless** [comment]
Yes me I have exact same issue...

**Comment by Sen** [comment]
I have same issue too, any solution?

**Comment by ssoulless** [comment]
Well a quick fix is to downgrade to 1.4, I had an urgency then I did that.

If you are using home-brew do the following:
```
brew uninstall elasticsearch
```
```
brew install homebrew/versions/elasticsearch14
```
It will install the previous version of elastic search that does not have this issue, then you can start elastic search without any problem:
```
launchctl load ~/Library/LaunchAgents/homebrew.mxcl.elasticsearch.plist
```

**Comment by Sen** [comment]
@ssoulless brew just rollbacked to 1.73, fixing local elasticsearch formula file works too

**Comment by zappeee** [comment]
I have the same problem, though I've installed via apt-get / dkpg.

**Comment by nik9000** [comment]
You could just stop using the parameter.... It's no longer supported
because the config dir option gives you as much flexibility in a slightly
different way and it is much simpler for us to support.

**Comment by zappeee** [comment]
I would if I just knew where it was used :)

I did a find on my machine and cannot find it anywhere, so any help would be appreciated!

**Comment by jasontedor** [comment]
It's coming from the p-list that the Homebrew formula is building. I'll open up a pull request in the Homebrew project to fix several issues that I see with the formula.

**Comment by nik9000** [comment]
@zappeee, it looks like you aren't using homebrew - if you want to contact me directly I can connect with you outside the ticket and we can track down where the setting comes from for you. My email is in my github profile.

**Comment by jasontedor** [comment]
I've opened Homebrew/homebrew#45644 to address this and other issues. Thank you for bringing them to our attention.

**Comment by jasontedor** [comment]
Closed by Homebrew/homebrew#45644.
